### PR TITLE
LG-9 Remove Old Calendar Items in Search

### PR DESCRIPTION
**Pull request by ChrispyAPAX**
- added ultimate cron module
  - added additional search fields to search API
  - modified solr search view filters with date and time filter and recurring event filter

### CODE REVIEW

File: sites/development.services.yml [hunk 1]
```diff
@@ -3,9 +3,6 @@
 # To activate this feature, follow the instructions at the top of the
 # 'example.settings.local.php' file, which sits next to this file.
 parameters:
-  twig.config:
```
Comment (chris-allen) [inline sites/development.services.yml:6]: Why was this removed?

**Review by chris-allen — CHANGES_REQUESTED** [review]

File: .gitattributes [hunk 2]
```diff
@@ -59,7 +59,3 @@
 *.ttf     -text diff
 *.woff    -text diff
 *.woff2   -text diff
-
-# Exclude generated files from PR diffs
```
[the remark this answers is not in the document]
Comment (ChrispyAPAX) [reply .gitattributes:63]: I swear I didn't touch this

File: sites/development.services.yml (hunk 1 above)
Comment (ChrispyAPAX) [reply sites/development.services.yml:6]: I swear I didn't touch this

File: .gitattributes (hunk 2 above)
[the remark this answers is not in the document]
Comment (TaylorAkin) [reply .gitattributes:63]: @ChrispyAPAX  It happened to me as well a few weeks ago. Not sure why mine removed the change, but it should be a one time thing. just git checkout dev <file path> for both of these

**Comment by ChrispyAPAX** [comment]
Alrighty I put those lines of code back the way they should be. Weird though - because I didn't touch those files. and I don't know what I did that would cause those changes to happen.

I have also made the changes @chris-allen requested for the filters

File: .gitattributes [hunk 3]
```diff
@@ -60,6 +60,7 @@
 *.woff    -text diff
 *.woff2   -text diff
 
+
```
Comment (TaylorAkin) [inline .gitattributes:63]: @ChrispyAPAX Can you just run git checkout dev .gitattributes? So there isn't a diff for this file?

Comment (ChrispyAPAX) [reply .gitattributes]: done!

**Comment by ChrispyAPAX** [comment]
OK. I was able to created a better defined filter to make sure that we are capturing all old events (I threw in meeting notices as well since it makes sense to filter those out too). It appears to work as intended on my machine. please check yourselves.

**Review by chris-allen — COMMENTED** [review]
The ticket title looks good to go. I question why ultimate cron is being introduced. I haven't  inspected what's been changed from the default configuration.

**Review by TaylorAkin — APPROVED** [review]

**Comment by ChrispyAPAX** [comment]
Ultimate cron was something we wanted for managing cron tasks in the future - it's not setup to be used right now , it's just installed.

**Review by chris-allen — APPROVED** [review]
